Flutter/shelter_partner/lib/repositories/auth_repository.dart

```diff
@@ -158,6 +158,7 @@ class AuthRepository {
       appendAnimalDataToURL: false,
       removeAds: true,
       simplisticMode: true,
+      locationTierCount: 2,
```

Comment:
I'd think 4 to match the current default
```suggestion
      locationTierCount: 4,
```